htdocs_symfony/src/Repository/CacheReportReasonsRepository.php

```diff
@@ -188,8 +190,6 @@ public function remove(GeoCacheReportReasonsEntity $entity)
             ['id' => $entity->id]
         );
 
-        $entity->cacheId = null;
-
         return $entity;
```

Comment:
So was muss drin bleiben sonst sagt dir das Objekt, dass der Eintrag noch in der Datenbank drin steht.

Reply:
Die Zeilen sind ist wieder drin und wurden korrigiert.